app/src/main/res/drawable/user_profile_card_background.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="rectangle">
+            <stroke android:color="#1D9EDA" android:width="2dp" />
```

Comment:
don't add colors in hardcoded form. Add color code in ``` color.xml ``` file

Reply:
okay done